### ISSUE TITLE
Minimum Atom version too low

### ISSUE THREAD

**Issue by cpearce31** [AtomLinter/linter-rubocop#314]
I teach a coding course and have 3 students using Ubuntu (two on 18.04, one on 18.10). I also use Ubuntu. We all upgraded to `linter-rubocop@2.2.4`, and stopped seeing any output from `linter-rubocop`. No errors in the console. Downgrading to 2.2.3 fixed it for all of us. Not sure how to go about troubleshooting, let me know if you need any more info.

**Comment by vzamanillo** [comment]
Hi! What rubocop version? Rvm or rbenv? Rubocop path?

**Comment by cpearce31** [comment]
Rubocop: 0.57.2
Rubocop Path: `/home/caleb/.rbenv/shims/rubocop`
Using rbenv with Ruby 2.5.0

Tried both with and without setting that path in the settings of linter-rubocop, confirmed that the path was right in `config.cson` when it was set in settings.

**Comment by vzamanillo** [comment]
I was trying to reproduce your case without any luck. The main difference between `v2.2.4` and `v2.2.3` is related to `activationHooks`, according to the symptoms you detail looks like the linter package does not fail silently, it is not getting activated, the question is why. What `Atom` version are you using?

**Comment by cpearce31** [comment]
Result of `atom --version`:
```
Atom    : 1.26.1
Electron: 1.7.11
Chrome  : 58.0.3029.110
Node    : 7.9.0
```

**Comment by vzamanillo** [comment]
Atom support for activation hooks by grammar scope is available from v1.29.0, you need to upgrade Atom to get the linter working.

**Comment by cpearce31** [comment]
OK, good to know, thank you!

**Comment by vzamanillo** [comment]
Be free to reopen the issue if you can get it working.

**Comment by Arcanemagus** [comment]
> Atom support for activation hooks by grammar scope is available from v1.29.0

The [minimum Atom version](https://github.com/AtomLinter/linter-rubocop/blob/4ec480180787a5a8944629789960166d8d22a7e1/package.json#L10) should be updated to account for this if that's the case @vzamanillo.

**Comment by Arcanemagus** [comment]
It [looks like](https://github.com/atom/atom/commit/d82c3979aca051acb86ee4fe6d0d339711c4acb7) the minimum version is actually Atom v1.30.0 as well.
